src/unix/fs.c

```diff
@@ -739,6 +734,45 @@ static int uv__fs_fstat(int fd, uv_stat_t *buf) {
 }
 
 
+typedef ssize_t (*uv__fs_buf_iter_yield)(uv_fs_t* req);
+static ssize_t uv__fs_buf_iter(uv_fs_t* req, uv__fs_buf_iter_yield yield) {
+  unsigned int iovmax;
+  unsigned int nbufs;
+  uv_buf_t* bufs;
+  ssize_t total;
+  ssize_t result;
+
+  iovmax = uv__getiovmax();
+  nbufs = req->nbufs;
+  bufs = req->bufs;
+  total = 0;
+
+  while (nbufs) {
+    req->nbufs = nbufs;
+    if (req->nbufs > iovmax) {
+      req->nbufs = iovmax;
+    }
```

Comment:
Two more style issues:

* libuv code is normally explicit about integral comparisons (i.e. `while (nbufs > 0)` instead of just `while (nbufs)`)

* one-liner control flow statements don't get braces